### DISCUSSION TITLE
More informative error message when function is missing

### DISCUSSION THREAD

**Discussion by liutiming** [ropensci/targets#413] [Ideas]
`Error: callr subprocess failed: argument "expr" is missing, with no default`

I was a bit silly today and forgot to put the function for one function, but the error message was not indicative where the error may be. 

Similar case when the target name is missing: `Error: callr subprocess failed: attempt to use zero-length variable name`

**Comment by wlandau** [comment]
Good point, informative errors should print as of a8cf19b64e69caff2739b2d98f08ee20119616ad.
